src/compas/geometry/pointcloud.py

```diff
@@ -57,6 +57,9 @@ def __init__(self, points, name=None):
     def __repr__(self):
         return "{0}(points={1!r})".format(type(self).__name__, self.points)
 
+    def __str__(self):
+        return "{0}(len(points)={1})".format(type(self).__name__, [point.__str__() for point in self.points])
```

Comment:
shouldn't this be having the `len(self.points)` instead? i.e.
```suggestion
        return "{0}(len(points)={1})".format(type(self).__name__, len(self.points))
```

Reply:
Thanks. I had half the intention there.